sdk/search/Azure.Search.Documents/apis.txt

```diff
@@ -31,34 +31,34 @@ namespace Azure.Search.Documents
 
         Response<AutocompleteResults> Autocomplete(string, string, AutocompleteOptions, CancellationToken) { }
         Task<Response<AutocompleteResults>> AutocompleteAsync(string, string, AutocompleteOptions, CancellationToken) { }
-        
+
         Response<IndexDocumentsResult> IndexDocuments(IndexDocumentsBatch<SearchDocument>, IndexDocumentsOptions, CancellationToken) { }
         Task<Response<IndexDocumentsResult>> IndexDocumentsAsync(IndexDocumentsBatch<SearchDocument>, IndexDocumentsOptions, CancellationToken) { }
         Response<IndexDocumentsResult> IndexDocuments<T>(IndexDocumentsBatch<T>, IndexDocumentsOptions, CancellationToken) { }
         Task<Response<IndexDocumentsResult>> IndexDocumentsAsync<T>(IndexDocumentsBatch<T>, IndexDocumentsOptions, CancellationToken) { }
     }
 
-    public class FieldBuilder { }
-    // TODO: FieldBuilder's attributes like [Key], [Searchable], etc.
+    public class SearchIndexBuilder { }
+    // TODO: SearchIndexBuilder's attributes like [Key], [Searchable], etc.
 
     public class SearchServiceClient
     {
+        SearchClient GetSearchClient(string) { }
         SearchIndexClient GetSearchIndexClient(string) { }
-
-
-
-
-
+        SearchDataSourceClient GetSearchDataSourceClient() { }
+        SearchIndexerClient GetSearchIndexerClient(string) { }
+        SearchSkillsetClient GetSearchSkillsetClient(string) { }
+        SearchSynonymMapClient GetSearchSynonymMapClient(string) { }
 
         Response<SearchServiceStatistics> GetServiceStatistics(SearchRequestOptions, CancellationToken) { }
         Task<Response<SearchServiceStatistics>> GetServiceStatisticsAsync(SearchRequestOptions, CancellationToken) { }
+    }
+}
 
-
-
-
-
-
-
+namespace Azure.Search.Documents.Indexes
+{
+    public class SearchDataSourceClient
+    {
         Response<DataSource> CreateDataSource(DataSource, SearchRequestOptions, CancellationToken) { }
```

Comment:
I didn't do this here, but we could probably drop the client name from all these APIs so this would become just `Create`, etc.